### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by fyzact**
fix how define dapr server to it can work directly when you copy

# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

- [x] Code compiles correctly
- [x] Created/updated tests
- [x] Extended the documentation

### CODE REVIEW

**Comment by shubham1172** [comment]
Hi, thanks for the PR, the code is self-explanatory there because of L104.

**Comment by fyzact** [comment]
> Hi, thanks for the PR, the code is self-explanatory there because of L104.

thanks for explanition, to me it was more aboout to fix the sample as it was not working directly when you copy.

**Comment by shubham1172** [comment]
Oh I just noticed the comma missing, thanks for pointing it out.

File: daprdocs/content/en/js-sdk-docs/js-server/_index.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ myApp.get("/my-custom-endpoint", (req, res) => {
 const daprServer = new DaprServer({
       serverHost: "127.0.0.1", // App Host
       serverPort: "50002", // App Port
-      serverHttp: myApp
+      serverHttp: myApp, //express server
```
Comment (shubham1172) [inline daprdocs/content/en/js-sdk-docs/js-server/_index.md:113]:
```suggestion
      serverHttp: myApp,
```

**Review by shubham1172 — APPROVED** [review]

**Comment by fyzact** [comment]
> Oh I just noticed the comma missing, thanks for pointing it out.

in fact, after pr I found too many issue like that, will fix all of them and raise new and more comprehensive pr next week hopefully.

**Comment by shubham1172** [comment]
That would be awesome, thanks a lot!

**Pull request merged**